src/pages/workspaces/workspace/Tools.js

```diff
@@ -79,6 +79,15 @@ const styles = {
   }
 }
 
+const sortTokens = {
+  lowerCaseName: notebook => notebook.name.toLowerCase()
```

Comment:
This variable name should probably be `config`, not `notebook`

Reply:
fixed, thanks!